extensions-core/hdfs-storage/pom.xml

```diff
@@ -355,6 +355,16 @@
         <artifactId>hadoop-hdfs-client</artifactId>
         <scope>runtime</scope>
       </dependency>
+      <dependency>
+        <groupId>org.codehaus.jackson</groupId>
+        <artifactId>jackson-jaxrs</artifactId>
+        <version>1.9.13</version>
```

Comment:
Please specify versions using dependencyManagement in the central pom. (It makes sure we use the same version everywhere)

Reply:
Added `codehause.jackson.version` in the central pom